website/docs/style/illustration/components/illustration-group.jsx

```diff
@@ -0,0 +1,193 @@
+import * as ReactDOM from 'react-dom';
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+
+import { Row } from '@semcore/grid';
+import OutsideClick from '@semcore/outside-click';
+import Copy from '@components/Copy';
+import Button from '@semcore/button';
+import FileDownloadM from '@semcore/icon/FileDownload/m';
+import CopyM from '@semcore/icon/Copy/m';
+
+const Section = styled.div`
```

Comment:
for next changes: It's much better to use css modules instead of styled components due to css-in-js performance issues